examples/diagnostics/compare_diags.py

```diff
@@ -148,21 +178,31 @@ def print_totals_and_diffs(config, refdata, devdata, varlist):
         )
         f = open(pathname, 'w')
 
-    # If we are printing only variables with zero diffs,
+
+    # Percent threshold for reporting differences
+    threshold = config["options"]["totals_and_diffs"]["small_diff_threshold"]
+
+    # If we are skipping variables with small differences,
     # then alert the user (print to screen & file)
-    if config["options"]["totals_and_diffs"]["skip_zero_diffs"]:
-        line = "... Only showing variables with nonzero differences"
+    if config["options"]["totals_and_diffs"]["skip_small_diffs"]:
+        diff_label = " |absolute difference|  > " + str(threshold)
+        if do_percent_diff:
+            diff_label = "|percent difference| > " + str(threshold) + " %"
+        line = "... Only showing variables with " + diff_label
```

Comment:
I personally like using [fstrings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python) for string concatenation. eg:
`diff_label = f"|percent difference| > {threshold}%"`